string-constants-lib/string-constants/private/danish-string-constants.rkt

```diff
@@ -933,6 +933,54 @@ please adhere to these guidelines:
   (experimental-languages "Eksperimentelle sprog")
   (initial-language-category "Startsprog")
   (no-language-chosen "Intet sprog valgt")
+  (other-languages "Andre sprog")
+
+  (module-language-name "Bestem sprog ud fra kildetekst")
+  (module-language-one-line-summary "#lang-linjen bestemmer det aktuelle sprog")
+  (module-language-auto-text "Automatisk #lang-linje") ;; shows up in the details section of the module language
+  ;; the next four string constants show up in the REPL in DrRacket in the "Racket Language",
+  ;; which is also the "Determine language from source" language. They are put right after the name
+  ;; of the language from the "#lang" line at the beginning of the source file
+  (module-language-repl-no-annotations "")
+  (module-language-repl-debug-annotations ", med debugning")
```

Comment:
I think ", med fejlsøgningsinformation" sounds better.

Reply:
I agree. I picked "debugning" because it was already used in the translation. I'll go change them all to "fejlsøgning" or "fejlsøgningsinformation", as needed.